draft-ietf-ppm-dap.md

```diff
@@ -358,10 +358,27 @@ client could not report 10^6s or -20s.
 
 # Message Transport
 
-Communications between DAP entities are carried over HTTPS {{!RFC2818}}. HTTPS
-provides server authentication and confidentiality. When client authenticaiton
-is also required, the client uses the mechanism described in
-{{https-sender-auth}}.
+Communications between DAP participants are carried over HTTPS {{!RFC2818}}.
+HTTPS provides server authentication and confidentiality. Use of HTTPS is
+REQUIRED.
+
+## Request Authentication {#request-authentication}
+
+DAP is made up of several sub-protocols in which different subsets of the
+protocol's participants interact with each other.
+
+In those cases where a channel between two participants is tunneled through
+another protocol participant, DAP mandates the use of authenticated encryption
+using {{!HPKE=RFC9180}} to ensure that only the intended recipient can see a
+message in the clear.
+
+In other cases, DAP requires client authentication. Any authentication scheme
```

Comment:
We should try to avoid overloading "Client" wherever possible.

```suggestion
In other cases, DAP requires sender authentication. Any authentication scheme
```